rqd/rqd/__main__.py

```diff
@@ -239,6 +249,10 @@ def main():
 
     setup_sentry()
 
+    # Start the watchdog process
+    start_watchdog()
```

Comment:
I like the idea of a watchdog service, but having it as a child process of rqd might mean it will get orphaned or killed when rqd exits, which defeats the purpose.

Maybe we could setup the systemd service configuration to restart automatically.

Try adding the following to the .service file:

```

[Service]
Restart=always
RestartSec=5

```

	•	Restart=always: This ensures the service is restarted whenever it exits, regardless of the exit code. Other options include on-failure, on-abnormal, etc., depending on the desired behavior.
	•	RestartSec=5: Specifies the number of seconds to wait before restarting the service. You can adjust this time as needed.

Reply:
Done!